cmd/kops/toolbox.go

```diff
@@ -18,14 +18,18 @@ package main
 
```

Comment:
Is toolbox the right place?  I see this option getting kinda big, and may even be a container running on k8s to push logs to s3.